CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+* Any `node_attr` in the tree can be used as a tip label, as well as the special-cases of strain-name and "none".
+  Previously we only allowed valid colorings.
+  ([#1668](https://github.com/nextstrain/auspice/pull/1668))
+* The specified tip label is surfaced more prominently within the the on-hover info boxes & on-click modals.
+  ([PR #1668](https://github.com/nextstrain/auspice/pull/1668))
```

Comment:
Keep consistent:

```suggestion
* Any `node_attr` in the tree can be used as a tip label, as well as the special-cases of strain-name and "none".
  Previously we only allowed valid colorings.
  ([#1668](https://github.com/nextstrain/auspice/pull/1668))
* The specified tip label is surfaced more prominently within the the on-hover info boxes & on-click modals.
  ([#1668](https://github.com/nextstrain/auspice/pull/1668))
```

Reply:
Fixed in [force push](https://github.com/nextstrain/auspice/compare/962a2e24e46a5a6f9b28a9157deebd662d9d08db..399c50181ab1ed35a88c1a6abc95b91193502eb0)